src/status_im2/contexts/chat/composer/constants.cljs

```diff
@@ -19,6 +19,8 @@
 
 (def ^:const images-container-height 76)
 
+(def ^:const link-preview-height 76)
+
```

Comment:
Nitpick: `link-preview-height` could be renamed to `links-container-height` for consistency